messaging/src/test/java/org/axonframework/tracing/SpanUtilsTest.java

```diff
@@ -67,4 +68,15 @@ void determineMessageNameForCommandWithName() {
                                                                             "SuperCommand");
         assertEquals("SuperCommand", SpanUtils.determineMessageName(message));
     }
+
+    @Test
+    void determineMessageNameForDeadlineWithoutPayload() {
+        GenericDeadlineMessage<String> message = new GenericDeadlineMessage<>("myDeadlineName");
+        assertEquals("myDeadlineName", SpanUtils.determineMessageName(message));
+    }
```

Comment:
Super nit: I'd expect a empty line here.
```suggestion
    }
    
```